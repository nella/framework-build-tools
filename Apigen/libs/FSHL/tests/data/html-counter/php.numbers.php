<?= echo -1.5 - 0.5 + -.5 + 20e6 + 20.20E6 + 20e-6 + 20.20E-6 + 20e+6 + 20.20E+6 + 0b100; ?>