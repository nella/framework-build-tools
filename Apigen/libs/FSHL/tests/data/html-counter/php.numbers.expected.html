<html><head><meta http-equiv="content-type" content="text/html; charset=utf-8" /><link rel="stylesheet" type="text/css" href="../../../style.css" media="all" /></head><body><pre>
<span class="line">1: </span><span class="xlang">&lt;?=</span> <span class="php-keyword1">echo</span> -<span class="php-num">1.5</span> - <span class="php-num">0.5</span> + -<span class="php-num">.5</span> + <span class="php-num">20e6</span> + <span class="php-num">20.20E6</span> + <span class="php-num">20e-6</span> + <span class="php-num">20.20E-6</span> + <span class="php-num">20e+6</span> + <span class="php-num">20.20E+6</span> + <span class="php-num">0b100</span>; <span class="xlang">?&gt;</span>
</pre></body></html>